pkg/component/load.go

```diff
@@ -204,6 +210,7 @@ func specFilesForDirectory(dir string) (map[string]Spec, error) {
 }
 
 // NewRuntimeSpecs creates a RuntimeSpecs from already loaded input and shipper runtime specifications.
+// Only used for testing.
```

Comment:
If it's only used for testing, is it possible to make it package private? Or we reference this in other packages ?

Reply:
The test that uses it is unfortunately in the `coordinator` package, so it can't be merged directly.